src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -17,6 +17,14 @@
 
 public class JavaDateTimeApi {
 
+    static final DateTimeFormatter formatDateFormatter = DateTimeFormatter.ofPattern(
```

Comment:
add private for all constants 
formatDateFormatter -> FORMAT_DATE_FORMATTER
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

Reply:
ready